docs/releases/1.11.txt

```diff
@@ -150,7 +150,8 @@ CSRF
 Database backends
 ~~~~~~~~~~~~~~~~~
 
-* ...
+* Added the ``skip_locked`` argument to :meth:`.QuerySet.select_for_update()`
+  on PostgreSQL and Oracle.
```

Comment:
PostgreSQL 9.5+
add something like: "to execute queries with FOR UPDATE SKIP LOCKED"